app/.eslintrc.json

```diff
@@ -17,6 +17,7 @@
   "plugins": ["@typescript-eslint", "jest"],
   "root": true,
   "rules": {
+    "@typescript-eslint/ban-ts-comment": "off",
```

Comment:
We turned this off for line 36 in _app.tsx because of a typing issue related to react context. We should resolve that issue and then we can remove this line.